docs/source/usage/wm-widgets.rst

```diff
@@ -44,12 +44,31 @@ Add the following to your :file:`~/.config/qtile/config.py`:
        ),
    ]
 
+.. _bar-usage:
+
+LemonBoy’s bar
+=============
```

Comment:
This bar is too short. ReST is not markdown, *its parser explicitly checks whether underline is long enough* and does not recognize too short bars as a headers.